src/cartridges/int_adyen_SFRA/cartridge/adyen/utils/adyenHelper.js

```diff
@@ -499,35 +537,18 @@ let adyenHelperObj = {
   },
 
   // creates a request object to send to the Adyen Checkout API
-  createAdyenRequestObject(order, paymentInstrument) {
+  createAdyenRequestObject(orderNo, orderToken, paymentInstrument) {
     const jsonObject = JSON.parse(paymentInstrument.custom.adyenPaymentData);
 
     const filteredJson = adyenHelperObj.validateStateData(jsonObject);
     const { stateData } = filteredJson;
 
-    let reference = 'recurringPayment-account';
-    let orderToken = 'recurringPayment-token';
-    if (order && order.getOrderNo()) {
-      reference = order.getOrderNo();
-      orderToken = order.getOrderToken();
-    }
-
-	// creates order number to be utilized for PayPal express
-	if (adyenHelperObj.isPayPalExpress(stateData.paymentMethod)){
-		const paypalExpressOrderNo = OrderMgr.createOrderNo();
-		session.privacy.paypalExpressOrderNo = paypalExpressOrderNo;
-		reference = paypalExpressOrderNo;
-	}
-
-    let signature = '';
     //Create signature to verify returnUrl if there is an order
-    if (order && order.getUUID()) {
-      signature = adyenHelperObj.createSignature(
+    let signature = adyenHelperObj.createSignature(
         paymentInstrument,
-        order.getUUID(),
-        reference,
+        UUIDUtils.createUUID(),
```

Comment:
UUIDUtils.createUUID() is used so that signature is consistent for express and express flows as in case of express flows we don't have an order yet. Also since the signature is never decoded it servers it purpose as long as the UUID added to the signature is unique.